demo/index.html

```diff
@@ -112,6 +113,17 @@ <h1>Home</h1>
 								<h1>About</h1>
 							</lit-route>
 							<lit-route path="/contact" component="my-contact"></lit-route>
+							<lit-route path="/lazy">
```

Comment:
do we need this route if we just put the link in the navbar?

Reply:
No we don't but I thought of adding a page with additional description of what was happening in the demo. I know it's obvious to us, but someone just picking it up might not know. If you want to just have the navbar load the component that's fine.